### DISCUSSION TITLE
Admin setup

### DISCUSSION THREAD

**Discussion by GWStudios-YT** [EvanMulawski/BeamMP.EnvSync#22] [Q&A]
I'm confused on how to do this am I doing anything wrong BTW the wiki doesn't help
![hjgkghjk](https://user-images.githubusercontent.com/70766404/224846750-da0cdb3b-b379-4019-b468-c9fbfaa93204.PNG)
![ghffgh](https://user-images.githubusercontent.com/70766404/224846753-ea1eff01-a15e-400d-b415-93578fbb2293.PNG)

**Comment by EvanMulawski** [comment]
Hi @GWRStudio, the `--` at the start of a line means the line is a comment. Remove the `--` in front of the `options.admins...` line.

Edit: missed the top part of the file. Keep `options.admins = {}` and only update the lines under the PERMISSIONS section.

**Reply by EvanMulawski** [reply]
Updated the wiki to be more specific.
